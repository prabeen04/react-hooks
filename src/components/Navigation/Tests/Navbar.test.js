import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Navbar from '../Navbar';

describe('<Navbar />', () => {
    describe('render()', () => {
        test('renders the component', () => {
            const wrapper = shallow(<Navbar />);
            const component = wrapper.dive();

            expect(toJson(wrapper)).toMatchSnapshot();
        });
    });
});