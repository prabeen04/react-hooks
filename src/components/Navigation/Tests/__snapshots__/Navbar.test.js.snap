// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> render() renders the component 1`] = `
<SearchAppBar
  classes={
    Object {
      "grow": "SearchAppBar-grow-2",
      "inputInput": "SearchAppBar-inputInput-8",
      "inputRoot": "SearchAppBar-inputRoot-7",
      "menuButton": "SearchAppBar-menuButton-3",
      "root": "SearchAppBar-root-1",
      "search": "SearchAppBar-search-5",
      "searchIcon": "SearchAppBar-searchIcon-6",
      "title": "SearchAppBar-title-4",
    }
  }
/>
`;
